pkg/tenant/ensign.go

```diff
@@ -64,18 +64,10 @@ func (s *TopicSubscriber) Subscribe() {
 		sub *sdk.Subscription
 	)
 
-	// Wait until the Ensign server is ready
-	// TODO: This should block Tenant from starting rather than waiting in a separate
-	// go routine.
-	if attempts, err := s.client.WaitForReady(); err != nil {
-		// Note: Using WithLevel with FatalLevel does not exit the program but this is
-		// likely a critical configuration error that we want to fix immediately.
-		log.WithLevel(zerolog.FatalLevel).Int("attempts", attempts).Err(err).Msg("could not connect to ensign server")
-		return
-	}
-
 	// Subscribe to the meta topic
 	if sub, err = s.client.Subscribe(); err != nil {
+		// Note: Using WithLevel with FatalLevel does not exit the program but this is
```

Comment:
Thanks for including this comment! Will a story need to be created for this?

Reply:
There's no TODO item, mainly just a reminder for if we encounter this during deployment.